src/api-service/__app__/notifications/__init__.py

```diff
@@ -15,9 +15,6 @@
 
 def get(req: func.HttpRequest) -> func.HttpResponse:
     entries = Notification.search()
-    for entry in entries:
-        entry.config.redact()
```

Comment:
We no longer need to do this, because it is handled by the new `SecretData` class.

Reply:
(Er, this is a comment to myself as I review, not an assertion that you need to do change anything)